src/frontend/packaging_info.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+/*
+As part of a Google Cloud demo, you can run an additional "packaging" microservice (HTTP server).
+This file contains code related to the frontend and the "packaging" microservice.
+*/
+
+type PackagingInfo struct {
+	Weight float32 `json:"weight"`
+	Width  float32 `json:"width"`
+	Height float32 `json:"height"`
+	Depth  float32 `json:"depth"`
+}
+
+func isPackagingServiceConfigured() bool {
+	return os.Getenv("PACKAGING_SERVICE_URL") != ""
+}
+
+func httpGetPackagingInfo(productId string) (*PackagingInfo, error) {
+	// Make the GET request
+	url := os.Getenv("PACKAGING_SERVICE_URL") + productId
+	fmt.Println("Requesting packaging info from URL: ", url)
+	resp, err := http.Get(url)
```

Comment:
@NimJay I agree with the scanner :slightly_smiling_face: _ANY_ URL can be placed here. While it is up to the cluster admin what the value would be but can we improve it a little by minimizing the attach surface? For example, can we limit URLs only to the cluster IPs or do another kind of validation?

Reply:
The demo will either use a:
1. public IP address for the packaging service
2. K8s `Service` address of the packaging service (`default.packagingservice`) — if the new packaging service ends up being in the same cluster (e.g., we might use Cloud Run instead of GKE for the packaging service)